update.sh

```diff
@@ -13,6 +13,9 @@ RASPBERRYPI_ARCHIVE_KEY_FILE_NAME="raspberrypi.gpg.key"
 RASPBERRYPI_ARCHIVE_KEY_URL="${RASPBERRYPI_ARCHIVE_KEY_DIRECTORY}/${RASPBERRYPI_ARCHIVE_KEY_FILE_NAME}"
 RASPBERRYPI_ARCHIVE_KEY_FINGERPRINT="CF8A1AF502A2AA2D763BAE7E82B129927FA3303E"
 
+FIRMWARE_PACKAGE=${RASPBERRYPI_ARCHIVE_KEY_DIRECTORY}/pool/main/f/firmware-nonfree/firmware-brcm80211_0.43+rpi5_all.deb
```

Comment:
My bad, this was from the first commit. In the second it was replaced with a general mechanism.